ballerina/llm_client_azure_open_ai.bal

```diff
@@ -69,10 +91,18 @@ isolated distinct client class AzureOpenAIModel {
             return error("No completion choices");
         }
 
-        string? resp = choices[0].message?.content;
-        if resp is () {
+        ChatCompletionMessageToolCalls? toolCalls = choices[0].message?.tool_calls;
+        
+        if toolCalls is () {
             return error("No completion message");
         }
-        return parseResponseAsType(resp, expectedResponseTypedesc);
+
+        string? resp = toolCalls[0].'function.arguments;
+
+        if resp is () || resp == RESPONSE_WITH_EMPTY_PARAMETERS  {
```

Comment:
"{}" is not no response? Why are we special casing this?

Reply:
Sometime it gives `{}` when we expecting a correct result.
Will handle that as a conversion error